pkg/cloudprovider/vsphereparavirtual/vmoperator/client/client.go

```diff
@@ -0,0 +1,66 @@
+package client
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/rest"
+
+	vmopv1alpha1 "github.com/vmware-tanzu/vm-operator-api/api/v1alpha1"
+
+	"k8s.io/cloud-provider-vsphere/pkg/cloudprovider/vsphereparavirtual/vmoperator"
+)
+
+var (
+	// VirtualMachineServiceGVR has virtualmachineservice resource info.
+	VirtualMachineServiceGVR = schema.GroupVersionResource{
+		Group:    "vmoperator.vmware.com",
+		Version:  "v1alpha1",
+		Resource: "virtualmachineservices",
+	}
+	// VirtualMachineGVR has virtualmachine resource info.
+	VirtualMachineGVR = schema.GroupVersionResource{
+		Group:    "vmoperator.vmware.com",
+		Version:  "v1alpha1",
+		Resource: "virtualmachines",
+	}
+)
+
+// VmoperatorV1alpha1Client contains the dynamic client for vm operator group
+type VmoperatorV1alpha1Client struct {
```

Comment:
Do we want to add a v1alpha1 path like `vmoperator/client/v1alpha1/client.go`? If we cut over to v1alpha2 vm-operator API, maybe we need to create another path `vmoperator/client/v1alpha1/client.go`.

Reply:
Yeah, it makes sense, will change to that way